src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -41,6 +41,20 @@ extern "C"
 
 using namespace OpenRCT2;
 
+constexpr uint32 SCENARIO_REPOSITORY_MAGIC_NUMBER = 0x58444953;
```

Comment:
Just out of curiosity (for the sake of self learning), what is the point of this magic number? did you randomly choose "58444953"? or is it tied to something else? If it's to keep stuff unique (similar to how the networkversion is used to connect to only identical versions), what is the difference between updating a magic number and just updating the repository version?

Reply:
This is hex for `XDIS`. What `XDIS` stands for, I do not know.

But it's common practice to identify files with some magic numbers, usually in text form, so that it can be more easily identified.

Reply:
Needs a comment, I'd say.